Resources/Lang/en_US.json

```diff
@@ -3614,5 +3614,6 @@
   "WardenIncreaseSpeed": "Increase Speed By",
   "WardenWarn": "DANGER! RUN!",
 
-  "MinionAbilityTime": "Ability Duration"
+  "MinionAbilityTime": "Ability Duration",
+  "blind": "blinded"
```

Comment:
Can you make `Minion_Blind`

Reply:
I mean sure I'll do it rn, but the reason I simply called it "blind" is hence it is a generic attribute which could potentially have use in the future for smt else (smt like what happened with arrogance/reverie)